### DISCUSSION TITLE
getting started with Carbon language

### DISCUSSION THREAD

**Discussion by himanshusingh0905** [carbon-language/carbon-lang#2663] [General]
Could anyone please suggest to me some documentation or something to get started with carbon language and start contributing to it?

**Comment by CelineausBerlin** [comment]
Hello Himanshu! Welcome to Carbon. :) 
I believe this is what you're looking for: https://github.com/carbon-language/carbon-lang/blob/trunk/CONTRIBUTING.md
Does that help?

**Reply by himanshusingh0905** [reply]
Thank you so much, ma'am.
But I meant what resources should I go through so that I can read them and start contributing

**Reply by jonmeow** [reply]
Hi! The CONTRIBUTING file contains a lot of important and helpful information. For example, you must sign the CLA if you want to merge PRs. As noted in the CONTRIBUTING file, only original content from you and other people who have signed the CLA can be accepted as Carbon contributions.

That said, the best way to find resources is to browse the README.md files. Our documentation is largely committed to GitHub, and available under the "docs" directory. Note the "Ways to contribute" section of the CONTRIBUTING file also includes several links you may find helpful.

**Reply by himanshusingh0905** [reply]
okay 
Thank you very much
